autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingTriggerFlowStatusHandler.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.periscope.monitor.handler;
+
+import java.util.Objects;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.periscope.domain.Cluster;
+import com.sequenceiq.periscope.monitor.event.ScalingTriggerFlowStatusEvent;
+import com.sequenceiq.periscope.service.ClusterService;
+
+@Component
+public class ScalingTriggerFlowStatusHandler implements ApplicationListener<ScalingTriggerFlowStatusEvent> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScalingTriggerFlowStatusHandler.class);
+
+    @Inject
+    private ClusterService clusterService;
+
+    @Override
+    public void onApplicationEvent(ScalingTriggerFlowStatusEvent event) {
+        long clusterId = event.getClusterId();
+        Cluster cluster = clusterService.findById(clusterId);
+        if (Objects.isNull(cluster)) {
+            LOGGER.info("No cluster found with id: {}", clusterId);
+            return;
+        }
+
+        // TODO: Determine how and what scalingTriggers need to be updated here.
```

Comment:
Let's discuss these separately.
@sodre90  - any thoughts on whether adding a batch API for looking up flowId information is a good approach, or alternate approaches for the same.
Context: Periscope is going to start storing flowIds for each trigger - and then tracks the status of these flows for success/failure and execution time. The question is whether to make individual API calls for each flowId to be tracked, or introduce a bulk API for the same, or something completely different.